lib/store_attribute/active_record/store.rb

```diff
@@ -167,7 +167,9 @@ def _define_accessors_methods(store_name, *keys, prefix: nil, suffix: nil) # :no
             end
 
             define_method(accessor_key) do
-              read_store_attribute(store_name, key)
+              next_value = read_store_attribute(store_name, key)
+              clear_attribute_change(store_name) unless changed?
```

Comment:
It seems that this change is redundant (or we don't have a test which proves it's required 🙂).

Also, if we still need it, shouldn't we check for this particular store attribute changed only? Currently, we consider all the attributes changes.

Reply:
Hmm let me take another look. I think the `changed?` is what calls `changed_in_place?` which allows us to do the comparison against the raw DB value.

You might be right about clearing the entire store though, I'll add a test where we change a couple of attributes.